app/src/main/java/com/github/swent/echo/compose/myevents/MyEventsScreen.kt

```diff
@@ -47,77 +28,31 @@ fun MyEventsScreen(myEventsViewModel: MyEventsViewModel, navActions: NavigationA
     ) {
-        // Will be adapted with the future viewModel
         val joinedEventsList by myEventsViewModel.joinedEvents.collectAsState()
         val createdEventsList by myEventsViewModel.createdEvents.collectAsState()
-        var myEventsTab by remember { mutableStateOf(MyEventsTab.JOINED_EVENTS) }
-
-        Column(modifier = Modifier.padding(it)) {
-            Row(modifier = Modifier.fillMaxWidth()) {
-                val paddingItems = 2.dp
-                Box(
-                    modifier =
-                        Modifier.weight(1f)
-                            .padding(paddingItems)
-                            .height(50.dp)
-                            .clickable { myEventsTab = MyEventsTab.JOINED_EVENTS }
-                            .testTag("my_events_joined_events_tab")
-                ) {
-                    Text(
-                        stringResource(R.string.my_events_joined_events),
-                        modifier = Modifier.align(Alignment.Center),
-                        style = MaterialTheme.typography.titleMedium,
-                    )
-                }
-                Box(
-                    modifier =
-                        Modifier.weight(1f)
-                            .padding(paddingItems)
-                            .height(50.dp)
-                            .clickable { myEventsTab = MyEventsTab.CREATED_EVENTS }
-                            .testTag("my_events_created_events_tab")
-                ) {
-                    Text(
-                        stringResource(R.string.my_events_created_events),
-                        modifier = Modifier.align(Alignment.Center),
-                        style = MaterialTheme.typography.titleMedium,
-                        textAlign = TextAlign.Center
-                    )
-                }
-            }
-            Box(modifier = Modifier.fillMaxWidth()) {
-                val underline = Modifier.height(1.dp).width(200.dp)
-                when (myEventsTab) {
-                    MyEventsTab.JOINED_EVENTS -> {
-                        Box(
-                            modifier =
-                                underline
-                                    .align(Alignment.CenterStart)
-                                    .padding(start = 10.dp)
-                                    .background(MaterialTheme.colorScheme.primary)
-                                    .testTag("my_events_underline_joined_events")
+        val isOnline by myEventsViewModel.isOnline.collectAsState()
+        Box(modifier = Modifier.padding(it)) {
+            Pager(
+                listOf(
+                    Pair(stringResource(R.string.my_events_joined_events)) {
+                        ListDrawer(
+                            joinedEventsList,
+                            "",
+                            "",
+                            isOnline,
+                            myEventsViewModel::refreshEvents
```

Comment:
[Nitpicking] Not sure if this is relevant to this PR, but it feels odd to pass empty string to the `ListDrawer` in order to (I assume) not show any section or semester. I believe it could be better (as in easier to understand for someone reading the code) to make both parameters in `ListDrawer` nullable, What do you think?
My assumptions might be wrong, so feel free to correct me.

Reply:
That's a good point ! Thanks for highlighting this. I will open an issue in order to change the way ListDrawer works, so don't have to pass those empty strings